stylesheet.css

```diff
@@ -1,9 +1,23 @@
 body {
-  padding-top: 0;
+  padding-top: 0rem;
   padding-left: 10rem;
+  font-family: 'futura';
+
+  /*background:linear-gradient(90deg,rgba(255,255,255,.1) 0,rgba(255,255,255,.1) 100%),url(images/wallpaper1.jpg);*/
+  background-image: url(images/wallpaper3.jpg);
+  background:linear-gradient(90deg,rgba(255,255,255,.1) 0,rgba(255,255,255,.1) 100%),url(images/wallpaper5.jpg)
+  
+  /*center and scale image nicely*/
+  background-position: center 0;
+  background-repeat: no-repeat;
+  background-size: 100% auto;
+
+
 }
 
+
 h1, h2, h3, h4, h5, h6 {
+  color: #FFF0F5;

```

Comment:
You can barely see the title colours are you going to change the background color as well?